beacon-chain/sync/subscriber_beacon_blocks.go

```diff
@@ -124,8 +137,9 @@ func (s *Service) reconstructAndBroadcastDataColumnSidecars(ctx context.Context,
 	// Broadcast and save data columns sidecars to custody but not yet received.
 	sidecarCount := uint64(len(sidecars))
 	for columnIndex := range info.CustodyColumns {
+		log := log.WithField("columnIndex", columnIndex)
 		if columnIndex >= sidecarCount {
-			log.WithField("index", columnIndex).Error("Sidecar index out of range - should never happen")
+			log.Error("Sidecar index out of range - should never happen")
```

Comment:
I'd rename it to "column custody index out of range"
Saying sidecar index here made me thing it's referring to `sidecars, err := s.cfg.executionReconstructor.ReconstructDataColumnSidecars`

Reply:
Fixed in 42fb59a9779280631bd054f1df2995b7143fb8a3.